lib/base-compiler.ts

```diff
@@ -1434,6 +1434,14 @@ export class BaseCompiler implements ICompiler {
             };
         }
 
+        if (output.truncated) {
+            return {
+                error: 'Clang exceeded max output limit',
```

Comment:
Should this read something other than "clang" since the opt pipeline stuff is now more generalized?

Reply:
Ah yeah, perhaps replace Clang with generic “compiler” or just remove this part of the message… I will do one of these next time I am at my desk. Let me know if you have a preference.

Reply:
Removed "Clang" from the message, hopefully that seems reasonable.